src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Ball {
+    static final int MAX_NUMBER = 100;
```

Comment:
I think It's better to move this constant in Lottery class(as we can have many lotteries in the future with different max numbers). And don't forget about access modifiers(if you don't use this constant outside class, where it defined, make it private)